demo/src/app/components/modal/demos/basic/modal-basic.ts

```diff
@@ -0,0 +1,21 @@
+import {Component} from '@angular/core';
+
+import {NgbModal} from '@ng-bootstrap/ng-bootstrap';
+
+@Component({
+  selector: 'ngbd-modal-basic',
+  templateUrl: './modal-basic.html'
+})
+export class NgbdModalBasic {
+  closeResult: string;
+
+  constructor(private modalService: NgbModal) {}
+
+  open(content) {
+    this.modalService.open(content).result.then((result) => {
+      this.closeResult = `Closed with: ${result}`;
+    }, (reason) => {
+      this.closeResult = `Dismissed with: ${reason}`;
```

Comment:
escape and backdrop click return `1` and `0` as dismiss reasons which are `NgbDismissReasons` values. But it looks strange in the demo → should switch over enum values and display text?